.github/get-feature-branch.php

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env php
+<?php
+
+error_reporting(E_ALL);
+set_error_handler(static function(int $errno, string $errstr, string $errfile = null, int $errline = null) {
+    throw new ErrorException($errstr, 0, $errno, $errfile, $errline);
+});
+
+$versions = json_decode(
+    file_get_contents('https://flex.symfony.com/versions.json'),
```

Comment:
This script should be removed in favor of using jq.
It's already used in the Travis file for inspiration.

Reply:
Done.